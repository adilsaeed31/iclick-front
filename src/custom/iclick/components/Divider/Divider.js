import React, { Component } from "react";
import PropTypes from "prop-types";

export default class Divider extends Component {
  static propTypes = {
    label: PropTypes.string
  }

  render() {
    const { label } = this.props;

    return (
      <div>
        <hr />
        {!!label && <div component="span">{label}</div>}
        <hr />
      </div>
    );
  }
}
