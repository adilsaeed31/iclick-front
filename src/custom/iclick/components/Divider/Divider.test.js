import React from "react";
import renderer from "react-test-renderer";
import { MuiThemeProvider } from "@material-ui/core/styles";
import theme from "custom/reactionTheme";
import Divider from "./Divider";

test("basic snapshot", () => {
  const component = renderer.create((
    <MuiThemeProvider theme={theme}>
      <Divider />
    </MuiThemeProvider>
  ));
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
