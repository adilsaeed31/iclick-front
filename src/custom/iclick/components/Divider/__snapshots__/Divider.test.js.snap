// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`basic snapshot 1`] = `
<div
  className="SkDivider-container-1"
>
  <hr
    className="SkDivider-item-3"
  />
  <hr
    className="SkDivider-item-3"
  />
</div>
`;
